UI/window-basic-settings-stream.cpp

```diff
@@ -255,7 +278,12 @@ void OBSBasicSettings::SaveStream1Settings()
 		obs_data_set_bool(settings, "bwtest", false);
 	}
 
-	obs_data_set_string(settings, "key", QT_TO_UTF8(ui->key->text()));
+	if (whip)
+		obs_data_set_string(settings, "bearer_token",
+				    QT_TO_UTF8(ui->key->text()));
+	else
+		obs_data_set_string(settings, "key",
+				    QT_TO_UTF8(ui->key->text()));
```

Comment:
This was already discussed and consensus was we should use `bearer_token`

Using the term `stream_key` adds significant tech debt/confusion as that doesn’t exist in WHIP.

Reply:
Answered in another conversations.

Reply:
@RytoEX @tt2468 @pkviet Can we keep this as 'Bearer Token'?

Key is never mentioned in any WHIP documentation. This would be confusing to users/implementors.